docs/detections/rules-ui-create.asciidoc

```diff
@@ -266,9 +266,16 @@ NOTE: To preview rules, you need the `read` privilege to the `.preview.alerts-se
 To preview a rule:
 
 . Write the rule query.
+. Choose how you want to preview the query results:
+
+** *Quick query preview*: Allows you to select from three pre-defined timeframes -- *Last hour*, *Last day*, or *Last month* -- when previewing rule results. By default, rules are scheduled to run every 5 minutes with an additional look-back time of 1 minute. These settings cannot be modified.
+** *Advanced query preview*: Allows you to choose a custom timeframe for the rule preview, schedule how often the rule should run, and specify a look-back time.
```

Comment:
We could mention that if user selects short rule interval while the timeframe is set to a long period, the warning message will appear indicating about potential preview execution timeout.

<img width="942" alt="Screenshot 2022-08-04 at 18 16 35" src="https://user-images.githubusercontent.com/2700761/182898056-27e53b63-ca98-4987-a9d9-8915909a7738.png">

Reply:
I incorporated this information as a tip under this step. Lemme know what you think when you have a chance @e40pud  - thanks!